local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deleter
+
+import (
+	"path/filepath"
+
+	"github.com/golang/glog"
+	"github.com/kubernetes-incubator/external-storage/local-volume/provisioner/pkg/types"
+
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+type Deleter interface {
+	// Cleanup and delete all its owned PVs that have been released
+	DeletePVs()
+}
+
+type deleter struct {
+	*types.RuntimeConfig
+}
+
+func NewDeleter(config *types.RuntimeConfig) Deleter {
+	return &deleter{RuntimeConfig: config}
+}
+
+func (d *deleter) DeletePVs() {
```

Comment:
If an admin removes a mount point, I would prefer that they go through the standard workflow of PV deletion instead of adding a new method.

You need admin privileges to create PVs, so I don't think we need to handle the case of a malicious admin.